src/ui/gclassroom/gclassroom.soy

```diff
@@ -0,0 +1,96 @@
+/**
+ * @fileoverview GClassroom gui for Coding in Chrome editor.
+ *
+ * @license Copyright 2018 The Coding with Chrome Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author efuquen@google.com (Edwin Fuquen)
+ */
+
+{namespace cwc.soy.GClassroom autoescape="strict"}
+
+
+/**
+ * Google Classroom template.
+ */
+{template .gClassroomTemplate}
+  {@param prefix: string}
+
+  <div id="{$prefix}body">
+    <div id="{$prefix}content">
+      <div id="{$prefix}course_list"></div>
+      <div id="{$prefix}course_work_list"></div>
+    </div>
+  </div>
+{/template}
+
+/**
+ * Google Classroom course list template.
+ */
+{template .gClassroomCourseListTemplate}
+  {@param prefix: string}
+  {@param courses: ?}
+  <div>
+    <h5>Select Course:</h5>
+    <ul class="mdl-list">
+      {foreach $course in $courses}
+      <li class="mdl-list__item {$prefix}course" data-id={$course['id']}>
```

Comment:
Should be surrounded by "..." and using a unique name like `data-course-id="{$course['id']}"`